src/custom-elements/src/components/carousel/carousel.scss

```
@@ -4,3 +4,21 @@
   display: block;
   width: 100%;
 }
+
+:host(chi-carousel[dots]) {
```

```suggestion
.chi-carousel {
  &.-dots {
    .chi-carousel__control {
      &.-next,
      &.-previous {
        height: $carousel-controller-height-dots; // calc(100% - 0.75rem)
        // height: $carousel-controller-height-pagination; // calc(100% - 1.25rem)
      }
    }
  }
```
`/src/chi/components/carousel.scss`